README.md

```diff
@@ -73,52 +73,47 @@ Here is a list of reasons how we improved on zap with slog.
+     slog also automatically prints a Go 1.13 error chain as an array. See screenshot above.
+     - zap logs multiline fields and errors stack traces as JSON strings which made them unreadable in a terminal.
 
-1. Lack of [context.Context](https://blog.golang.org/context) support.
+1. Full [context.Context](https://blog.golang.org/context) support
+   `slog` lets you set fields in a `context.Context` such that any log with the context prints those fields.
 
    - We wanted to be able to pull up all relevant logs for a given trace, user or request. With zap, we were plugging
      these fields in for every relevant log or passing around a logger with the fields set. This became very verbose.
-     - `slog` lets you set fields in a `context.Context` such that any log with the context prints those fields.
 
-1. zap is hard and confusing to extend. There are too many structures and configuration options.
+1. Simple and easy to extend
 
-   - With slog, the extension model is based on the single method Sink interface. Logger is the
-     concrete type around Sink used to provide the high level API.
+   - A new backend only has to implement the simple Sink interface.
+   - zap is hard and confusing to extend. There are too many structures and configuration options.
 
-1. We found ourselves often implementing zap's [ObjectMarshaler](https://godoc.org/go.uber.org/zap/zapcore#ObjectMarshaler)
-   to log Go structures.
+1. Automatic structured logging of Go structures (including private fields)
 
-   - This was very verbose and most of the time we ended up only implementing `fmt.Stringer` and using `zap.Stringer`
-     instead.
-   - slog handles Go structures transparently, including private fields. One
-     may implement [`slog.Value`](https://godoc.org/cdr.dev/slog#Value) to override the representation,
+   - One may implement [`slog.Value`](https://godoc.org/cdr.dev/slog#Value) to override the representation,
      use struct tags to ignore or rename fields and even reuse the
      [`json.Marshal`](https://golang.org/pkg/encoding/json/#Marshal) representation
      with [`slog.JSON`](https://godoc.org/cdr.dev/slog#JSON).
+   - With zap, We found ourselves often implementing zap's
+     [ObjectMarshaler](https://godoc.org/go.uber.org/zap/zapcore#ObjectMarshaler) to log Go structures. This was
+     very verbose and most of the time we ended up only implementing `fmt.Stringer` and using `zap.Stringer`
+     instead.
 
-1. We had many helper functions for logging but we wanted the line reported to be of the parent function.
-   zap has an [API](https://godoc.org/go.uber.org/zap#AddCallerSkip) for this but it's verbose and requires
-   passing the logger around explicitly.
+1. slog takes inspriation from Go's stdlib and implements [`slog.Helper`](https://godoc.org/cdr.dev/slog#Helper) which works just like
+   [`t.Helper`](https://golang.org/pkg/testing/#T.Helper) to mark the calling function as a helper and skip it when
+   reporting location info.
```

Comment:
This is a pretty long bullet point. Maybe just leave this as
```
slog takes inspiration from Go's stdlib
```
and move the rest to bullet points?

Reply:
Will fix.